setup.py

```diff
@@ -5,6 +5,14 @@
 
 from setuptools import setup
 
+install_requires = ['catkin_pkg', 'PyYAML']
+
+if (
+    sys.version_info[0] > 3 or
```

Comment:
Won't this need import sys?

Reply:
Line 4?

Reply:
My mistake, sys was neither imported nor used in the diff context.

Reply:
This is a follow up to #182 in that we have the conditional for if it's less that python 3.8. To that end should we have the same check here?

Reply:
@tfoote I don't understand the question. The condition checks exactly that: if the Python version is 3.8 or higher the `distro` dependency is needed.

Reply:
sorry I missed the second half under the comment